storage/tests/integration/conftest.py

```diff
@@ -5,7 +5,8 @@
 
 @pytest.fixture(scope='module')  # type: ignore
 def bucket_name() -> str:
-    return 'voiceai-staging-public-test'
+    return 'fststaging_nick_testing'
```

Comment:
Note to self: don't merge until this gets changed back.

Reply:
doh, totally meant to revert this, will do now.